os/drivers/testcase/testcase_proto.h

```diff
@@ -28,4 +28,7 @@ int test_sched(int cmd, unsigned long arg);
 int test_signal(int cmd, unsigned long arg);
 #endif
 int test_compress_decompress(int cmd, unsigned long arg);
+#ifdef CONFIG_AMEBAD_TRUSTZONE
```

Comment:
I think using `CONFIG_ARMV8M_TRUSTZONE` is much better. Because, in the future we can support more boards which have trustzone, not only amebad.

Reply:
rdp api's are defined only for amebad. So I have kept it under this flag. In future, if we add new vendor implementation or a generic TZ verification code, then we can modify this.